### PR TITLE
add whirl.codes to global blacklist

### PR DESCRIPTION
**Pull request by Whirl21**
add my site whirl.codes to global blacklist!

### CODE REVIEW

File: src/config/dark_sites.json [hunk 1]
```diff
@@ -79,6 +79,7 @@
     "undertale.com",
     "vakhtangov.ru",
     "vinesauce.com",
+    "whirl.codes",
```
Comment (Gusted) [inline src/config/dark_sites.json:82]: We don't use the dark_sites.json anymore.

**Comment by Whirl21** [comment]
removed

File: src/config/dark_sites.json [hunk 2]
```diff
@@ -79,6 +79,7 @@
     "undertale.com",
     "vakhtangov.ru",
     "vinesauce.com",
+
```
Comment (Gusted) [inline src/config/dark_sites.json:82]:
```suggestion
```

**Comment by Whirl21** [comment]
again removed 😳

**Review by Gusted — APPROVED** [review]

**Pull request merged**